rocky/katalogus/views/plugin_settings_list.py

```diff
@@ -14,7 +14,7 @@ class PluginSettingsListView(SinglePluginView):
     """
 
     paginator_class = RockyPaginator
-    paginate_by = 10
+    paginate_by = 150
```

Comment:
fine for now, we are going to show the entire schema as a single form i the future.